src/main/java/reposense/model/Summary.java

```diff
@@ -0,0 +1,24 @@
+package reposense.model;
+
+import java.util.List;
+
+/**
+ * Stores the summary.json file values.
```

Comment:
/**
 * Represents the structure of a summary.json in reposense-report folder.
 */